src/main/java/com/sap/sgs/phosphor/fosstars/model/score/oss/ProjectActivityScore.java

```diff
@@ -159,6 +193,26 @@ private static void check(Value<Integer> commits, Value<Integer> contributors) {
     }
   }
 
+  /**
+   * Reverts a number of values of a function on a specified interval and prints them out.
+   *
+   * @param func The function to be reverted.
```

Comment:
Hmm I am confused now, does this function revert or invert? because the name of the function is printInvertedValues.

Reply:
It should be "... to be inverted". I'll update it.